lnbits/core/services/payments.py

```diff
@@ -288,6 +289,13 @@ async def send_payment_notification(wallet: Wallet, payment: Payment):
         send_chat_payment_notification(wallet, payment)
     except Exception as e:
         logger.error("Error sending chat payment notification", e)
+    try:
+        await send_payment_push_notification(payment)
```

Comment:
should take `wallet` as an optional parameter

Reply:
fixed